docs/migrate.md

```diff
@@ -4,6 +4,11 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.8
+
+When you upgrade to v3.8 and you use namespaces make sure cluster role binding and service account have the same
```

Comment:
"Using namespaces" isn't accurate here. Every Kubernetes resources reside in a namespace (See https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/ for more general information). You can just choose to install Sourcegraph inside a namespace that isn't `default`. I would also refer to the section on non-default namespaces in configure.md.

Reply:
done